core/controllers/admin.py

```diff
@@ -606,6 +622,131 @@ def _create_dummy_skill(
         skill.update_explanation(state_domain.SubtitledHtml('1', explanation))
         return skill
 
+    def _load_dummy_blog_post(self, blog_post_title: str) -> None:
+        """Loads the database with a blog post
+
+        Raises:
+            Exception. Cannot load new blog post in production mode.
+            Exception. User does not have enough rights to generate data.
+        """
+        assert self.user_id is not None
+        if not constants.DEV_MODE:
+            raise Exception('Cannot load new blog post in production mode.')
+
+        blog_post = blog_services.create_new_blog_post(self.user_id)
+        fs = fs_services.GcsFileSystem('blog_post', blog_post.id)
+        with open(
+            './assets/images/general/learner1.png', 'rb'
+        ) as thumbnail:
+            fs.commit(
+                'thumbnail/blog_thumbnail.png',
+                thumbnail.read(),
+                'image/png'
+            )
+        with open(
+            './assets/images/subjects/Art.svg', 'rb'
+        ) as image:
+            fs.commit(
+                'image/blog_post_image_height_326_width_490.svg',
+                image.read(),
+                'image/svg+xml'
+            )
+        random_string = utils.generate_random_string(6)
+        try:
+            if blog_post_title == 'Education':
+                blog_services.update_blog_post(blog_post.id, {
+                    'title': 'Education-' + random_string,
+                    'thumbnail_filename': 'blog_thumbnail.png',
+                    'content': """
```

Comment:
Instead of inlining these big strings, could you define them as named constants at the top of the file and refer to those named constants here?

Reply:
Done.